src/test/java/bio/terra/service/duos/DuosClientTest.java

```diff
@@ -106,9 +147,85 @@ public void testGetDatasetThrowsWhenIdMalformed() {
         equalTo(expectedEx));
   }
 
+  @Test
+  public void testGetApprovedUsers() {
+    var expectedBody =
+        new DuosDatasetApprovedUsers(
+            List.of("a", "b", "c").stream().map(DuosDatasetApprovedUser::new).toList());
```

Comment:
You can create a stream directly from a list of objets
```suggestion
            Stream.of("a", "b", "c").map(DuosDatasetApprovedUser::new).toList());
```